frontend/src/metabase/collections/components/PinnedItemCard/PinnedItemCard.unit.spec.js

```diff
@@ -110,4 +128,29 @@ describe("PinnedItemCard", () => {
       );
     });
   });
+
+  describe("description", () => {
```

Comment:
I think since this is now handled by `MarkdownPreview` component, we should only have maybe 1 integration test here with a very basic case, and move advanced tests to the component itself

Reply:
02656114e633871245785e77bf494b836b7bc4d6